README.md

```diff
@@ -145,15 +146,18 @@ But there are two prerequisites:
 One is the need to set up passwordless ssh access.
 For the passwordless ssh access please follow [here](https://danidudas.medium.com/how-to-connect-to-raspberry-pi-via-ssh-without-password-using-ssh-keys-3abd782688a).
 
-Second is the need to set the autostart.
+Second is the need to set the [autostart](https://github.com/omarcostahamido/PVM#autostart).
 
-Once you're done, run the command on your computer.
+Once you're done, run the command on the control pc.
 
-`bash deploy_code_to_rpi.sh`
+```bash
+bash deploy_code_to_rpi.sh
 
-`cd PVM && git pull`
+# Download the latest code on RPIs with the IP address in host_ip.txt.
+cd PVM && git pull
 
-`sudo reboot`
+sudo reboot
```

Comment:
so you're saying that these 2 lines of code will be executed on the rpi devices even though they are typed on the control machine?
Ok, got it. It feel that these lines of code could very well be already part of the `deploy_code_to_rpi.sh` script no?

Reply:
Yes, you are right about the idea. The control PC can reboot all RPIs when typing `reboot` in its console.
I try to do it in the script. Unfortunately, pdsh will create a new interactive session that will not receive the command from the script. 
But I will still work on this if a better method comes up.

Reply:
Thank you @KaneBetter ! 😄